### PR TITLE
bitcoind: set default ports in regtest mode

### PR DESCRIPTION
**Pull request by jonasnick**

### CODE REVIEW

File: modules/bitcoind.nix
Comment (barbiek1st) [inline modules/bitcoind.nix:1]: jonasnick:regtest-default-ports

**Review by barbiek1st — APPROVED** [review]

**Review by barbiek1st — APPROVED** [review]

**Comment by erikarvstedt** [comment]
[Fixup](https://github.com/erikarvstedt/nix-bitcoin/commits/regtest-default-ports-ea/).

**Comment by erikarvstedt** [comment]
This shows the exact defaults in a doc view of the options (like manpages or NixOS options search).

**Comment by jonasnick** [comment]
Thx @erikarvstedt. I was wondering how doc viewers would handle a non-constant default.

**Comment by barbiek1st** [comment]
Hello I need full explanation about this process

**Review by erikarvstedt — APPROVED** [review]
ACK 143fe30e03495da06ff868480325b29073e0dfac

**Pull request merged**